src/gui/qgstextformatwidget.cpp

```diff
@@ -1869,6 +1879,25 @@ void QgsTextFormatWidget::updateDataDefinedAlignment()
                                     || mCoordPointDDBtn->isActive() );
 }
 
+void QgsTextFormatWidget::overlapModeChanged()
+{
+  QString description;
+  switch ( static_cast< Qgis::LabelOverlapHandling >( mComboOverlapHandling->currentData().toInt() ) )
+  {
+    case Qgis::LabelOverlapHandling::PreventOverlap:
+      description = tr( "Overlapping labels will never be placed for the layer, even if it means some labels will be missing." );
```

Comment:
We could maybe add a tip along the lines of "To see unplaced labels, use the show unplaced labels toolbar action" too there. No strong opinions.

Reply:
done